src/azure/ApiCenter/ApiCenterService.ts

```diff
@@ -19,6 +19,25 @@ export class ApiCenterService {
     this.resourceGroupName = resourceGroupName;
   }
 
+  public async isResourceGroupExist(): Promise<boolean> {
```

Comment:
better naming, e.g. `checkResourceGroupExists`?

Reply:
`isResourceGroupExist` is short and clear, is there any guidance that says `check***` is better?

Reply:
both are OK. If using `is`, then `Exist` would be adj. : `Existing`?